operator-pipeline-images/operatorcert/entrypoints/index.py

```diff
@@ -53,6 +53,12 @@ def setup_argparser() -> argparse.ArgumentParser:  # pragma: no cover
         default="",
         help="A tag suffix to append to the index image when copying to the destination.",
     )
+    parser.add_argument(
+        "--mode",
+        choices=["replaces", "semver", "semver-skippatch"],
+        default="",
```

Comment:
I think having the default as `None` would be a bit cleaner, but your call.

Reply:
Fixed